tomster-player/app/styles/app.scss

```diff
@@ -61,6 +61,26 @@ $playerHeight: 75px;
   }
 }
 
+$sidebarWidth: 200px;
+
+.sidebar {
+  float: left;
+  width: $sidebarWidth;
+
+  .album-cover {
+    width: 100%;
+  }
+
+  .rating, .song-list {
+    display: none;
```

Comment:
let's do this via an attribute on the `album-list` component, e.g. `{{album-list album=(readonly album) showSongs=false}}`